01-backup.md

```diff
@@ -35,12 +35,21 @@ Here's how Dracula sets up his new laptop:
 $ git config --global user.name "Vlad Dracula"
 $ git config --global user.email "vlad@tran.sylvan.ia"
 $ git config --global color.ui "auto"
-$ git config --global core.editor "nano"
 ~~~
 
-(Please use your own name and email address instead of Dracula's,
-and please make sure you choose an editor that's actually on your system,
-such as `notepad` on Windows.)
+(Please use your own name and email address instead of Dracula's.)
+
+He also has to set his favorite text editor, following this table:
+
+| Editor             | Configuration command                            |
+|:-------------------|:-------------------------------------------------|
+| nano               | `$ git config --global core.editor "nano -w"`    |
```

Comment:
On Mon, Feb 23, 2015 at 05:26:30PM -0800, Raniere Silva wrote:
> > +| nano               | `$ git config --global core.editor "nano -w"`    |
> 
> This works on Windows?

Yup, because our Windows installer installs nano [1] and puts it in
the PATH [2,3].  It looks like nano is installed on OS X too [4], and
it's installed by default on most Linux distributions I've seen.

[1]: https://github.com/swcarpentry/windows-installer/blob/v0.2/swc-windows-installer.py#L284
[2]: https://github.com/swcarpentry/windows-installer/blob/v0.2/swc-windows-installer.py#L287
[3]: https://github.com/swcarpentry/windows-installer/blob/v0.2/swc-windows-installer.py#L291
[4]: https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/nano.1.html